quinn-udp/src/unix.rs

```diff
@@ -795,6 +885,99 @@ fn decode_recv(
     })
 }
 
+#[cfg(target_os = "linux")]
+fn recv_err(io: SockRef<'_>) -> io::Result<Option<IcmpError>> {
+    use std::mem;
+
+    let fd = io.as_raw_fd();
+
+    let mut control = cmsg::Aligned([0u8; CMSG_LEN]);
+
+    // We don't need actual data, just the error info
+    let mut iovec = libc::iovec {
+        iov_base: std::ptr::null_mut(),
+        iov_len: 0,
+    };
+
+    let mut addr_storage: libc::sockaddr_storage = unsafe { mem::zeroed() };
+
+    // Have followed the previous declarations
```

Comment:
Would you mind changing this comment to something more clear, I'm not sure what it means